plugin.yaml

```diff
@@ -40,6 +40,16 @@ node_types:
                 A dictionary of parameters allowed by docker-py to the
                 create_container function.
               default: {}
+            daemon_client:
```

Comment:
but thinking about this further, if you want to use this with 3.2, why are you requesting to pull to master (which will be 3.3m5?) Let's make use of the 3.3 feature.